.devcontainer/docker-compose.yaml

```diff
@@ -12,6 +26,9 @@ services:
       OPA_URL: http://opa:8181
       DATABASE_URL: postgres://postgres:password@postgres
       RABBITMQ_URL: amqp://rabbitmq:password@rabbitmq
+      CRYSTAL_LIBRARY_PORT: 8000
+      COMPOUND_LIBRARY_PORT: 8001
+      COMPOUND_SOAKING_PORT: 8002
```

Comment:
Could we add ports for the other existing services? Iirc `targeting` and `pin_packing`